src/Xamarin.Android.Build.Tasks/Tasks/GenerateJavaStubs.cs

```diff
@@ -287,19 +284,13 @@ void SaveResource (string resource, string filename, string destDir, Func<string
 
 		void WriteTypeMappings (List<TypeDefinition> types)
 		{
-			using (var gen = UseSharedRuntime
-				? new TypeNameMapGenerator (types, Log.LogDebugMessage)
-			        : new TypeNameMapGenerator (ResolvedAssemblies.Select (p => p.ItemSpec), Log.LogDebugMessage)) {
```

Comment:
@jonpryor I bet this `UseSharedRuntime ?` check was important...

I need to look at what is in the typemaps in both cases.